bin/atomizer

```diff
@@ -70,9 +74,19 @@ function parseFile (file, recursive, dir) {
         stat = fs.statSync(filepath);
 
         if (stat.isFile()) {
-            if (!program.quiet) console.warn('Parsing file ' + chalk.cyan(relative) + ' for Atomic CSS classes');
-            fileContents = fs.readFileSync(filepath, {encoding: 'utf-8'});
-            classNames = atomizer.findClassNames(fileContents);
+            var isExcluded = false;
+            if (program.exclude) {
+              isExcluded = some(program.exclude, function excludeFile(value) {
+                return minimatch(filepath, value, { matchBase: true });
+              })
+            }
+            if (!isExcluded) {
+              if (!program.quiet) console.warn('Parsing file ' + chalk.cyan(relative) + ' for Atomic CSS classes');
+              fileContents = fs.readFileSync(filepath, {encoding: 'utf-8'});
+              classNames = atomizer.findClassNames(fileContents);
+            } else {
+              if (!program.quiet) console.warn('Excluding file ' + chalk.cyan(relative) + ' for Atomic CSS classes');
```

Comment:
Awesome, you picked up the `quiet` change I just merged 💯

Reply:
I think a better way is to just wrap `console.warn` and have one check, but I'm lazy ...

Reply:
Probably true, but this is fine :)